java/src/main/java/cucumber/runtime/java/ObjectFactoryLoader.java

```diff
@@ -53,6 +55,46 @@ public static ObjectFactory loadObjectFactory(ClassFinder classFinder, String ob
         return objectFactory;
```

Comment:
It's okay to use an early return statement. It's just outside of the scope of your changes but will make this section of the code easier to read.